tests/test_code_gen.py

```diff
@@ -664,6 +664,16 @@ def test_fstring_escaped_braces(self):
         '''
         self.assertSrcRoundtripsGtVer(source, (3, 6))
 
+    @unittest.skipUnless(sys.version_info >= (3, 8),
+                         "f-string debugging introduced in Python 3.8")
+    def test_fstring_debugging(self):
+        source = '''
+        x = f'{5=}'
+        y = f'{5=!r}'
+        z = f'{3*x+15=}'
+        '''
```

Comment:
Could you add tests for {x=:}, {x=:.2f}, and f'alpha α {pi=} ω omega'? These should cover most of the tests from the original commit.